packages/eslint-plugin/src/rules/no-duplicate-enum-values.ts

```diff
@@ -0,0 +1,73 @@
+import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/utils';
+import * as util from '../util';
+
+export default util.createRule({
+  name: 'no-duplicate-enum-values',
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Disallow duplicate enum member values',
+      recommended: false,
+      suggestion: true,
+    },
+    hasSuggestions: true,
+    messages: {
+      duplicateMember: 'Duplicate enum member with value {{value}}.',
```

Comment:
Hmm, this phrasing makes me think that the member is what's duplicated -- like having two members with the same keys. How about...

```suggestion
      duplicateValue: 'Duplicate enum member value {{value}}.',
```